enterprise/frontend/src/metabase-enterprise/embedding/components/InteractiveEmbeddingSettings.tsx

```diff
@@ -0,0 +1,107 @@
+import type { ChangeEvent } from "react";
```

Comment:
in this PR there are 2 new settings page
- static embedding settings page
- interactive embedding settings page.

All settings pages are mostly the same except they have a new toggle at the top of their page.

![Screenshot 2024-09-12 at 1 13 53 AM](https://github.com/user-attachments/assets/368c7057-d37f-4421-ae8b-e424575feb63)

Reply:
Tested locally, static embedding works. However, I cannot disabled Interactive Embedding inside the page toggle. I am using EE. Could you take a look?

![CleanShot 2567-09-12 at 17 33 24](https://github.com/user-attachments/assets/f098f25d-7f5d-4b93-a448-79429c83726a)

Reply:
@WiNloSt I see! Let me approve this then